pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -593,6 +593,12 @@ type ClusterAutoscaler struct {
 	// IgnoreTaints specifies a list of taint keys to ignore in node templates when considering to scale a node group.
 	// +optional
 	IgnoreTaints []string `json:"ignoreTaints,omitempty" protobuf:"bytes,10,opt,name=ignoreTaints"`
+	// NewPodScaleupDelay specifies how long CA should ignore newly created pods before they have to be considered for scale-up
+	// +optional
+	NewPodScaleupDelay *metav1.Duration `json:"newPodScaleupDelay,omitempty" protobuf:"bytes,11,opt,name=newPodScaleupDelay"`
+	// MaxEmptyBulkDelete specifies the maximum number of empty nodes that can be deleted at the same time (default: 10)
+	// +optional
+	MaxEmptyBulkDelete *int32 `json:"maxEmptyBulkDelete,omitempty" protobuf:"bytes,12,opt,name=maxEmptyBulkDelete"`
```

Comment:
```suggestion
	MaxEmptyBulkDelete *int32 `json:"maxEmptyBulkDelete,omitempty" protobuf:"varint,12,opt,name=maxEmptyBulkDelete"`
```
Please remove the protobuf tag, and run `make generate WHAT="protobuf"`. This yields a separate result compared to this PR.